### PR TITLE
Refactoring code to handle availability information for a given holding.

### PR DESCRIPTION
**Pull request by hectorcorrea**

### CODE REVIEW

**Comment by christinach** [comment]
Can someone update the commit with @kayiwa github email?

**Comment by hackartisan** [comment]
I'll update it.

File: spec/adapters/alma_adapter_spec.rb [hunk 1]
```diff
@@ -362,11 +362,23 @@
...
       expect(item[:in_temp_library]).to eq true
       expect(item[:temp_library_code]).to eq "online"
+      expect(item[:pickup_location_id]).to eq "pa"
+      expect(item[:pickup_location_code]).to eq "pa"
+
+      # Test an actual response. These values are not particularly meaningful, but to make sure we don't
+      # inadvertently change them when refactoring.
+      item_test = { barcode: "32101080920208", id: "23105104390006421", copy_number: "1", status: nil,
+                    on_reserve: nil, item_type: "Gen", pickup_location_id: "pa", pickup_location_code: "pa",
+                    location: "online$etasrcp", label: "ReCAP", in_temp_library: true, status_label: "Item in place",
```
Comment (christinach) [inline spec/adapters/alma_adapter_spec.rb:377]: Can we change pickup_location_id: "pa" to -> 299? currently "PA" => "299".

**Review by christinach — CHANGES_REQUESTED** [review]

File: spec/adapters/alma_adapter_spec.rb (hunk 1 above)
Comment (christinach) [reply spec/adapters/alma_adapter_spec.rb:377]: I don't think these ids will change https://github.com/pulibrary/voyager_helpers/blob/0ae062a25177187fb6b47ae19e29b45ea469acb0/lib/voyager_helpers/liberator.rb#L1325

Comment (hackartisan) [reply spec/adapters/alma_adapter_spec.rb:377]: Can we make a separate issue for this? I don't think it's a refactor concern.

Comment (christinach) [reply spec/adapters/alma_adapter_spec.rb:377]: a separate issue to change the value of the pickup_location_id in the test?

Comment (hackartisan) [reply spec/adapters/alma_adapter_spec.rb:377]: Will changing the test not necessitate a code change?

Comment (christinach) [reply spec/adapters/alma_adapter_spec.rb:377]: yes it will change. I will make a separate issue for that.

Comment (christinach) [reply spec/adapters/alma_adapter_spec.rb:377]: opened https://github.com/pulibrary/bibdata/issues/1233

**Review by christinach — APPROVED** [review]

File: spec/adapters/alma_adapter_spec.rb (hunk 1 above)
Comment (hackartisan) [reply spec/adapters/alma_adapter_spec.rb:377]: Thanks!

**Pull request merged**